mobile/test/common/integration/cds_integration_test.cc

```diff
@@ -14,10 +14,7 @@ using envoy::config::cluster::v3::Cluster;
 
 class CdsIntegrationTest : public XdsIntegrationTest {
 public:
-  CdsIntegrationTest() {
-    use_lds_ = false;
-    default_request_headers_.setScheme("http");
-  }
+  CdsIntegrationTest() { default_request_headers_.setScheme("http"); }
```

Comment:
Although it's safe for this, it's probably better to standardize on using `initialize()` like the rest of the integration tests instead of doing some initialization in the constructor to avoid issues when calling virtual functions in the constructor.